geometries/brian/M82pck.tpc

```diff
@@ -7,14 +7,14 @@ NH_TARGET_BODIES         += ( 9999000 )
 BODY9999000_POLE_RA        = (    0.    0.    0. )
 BODY9999000_POLE_DEC       = (   90.    0.    0. )
 BODY9999000_PM             = (  270.    0.    0. )
-BODY9999000_RADII          = (  946073047258080.  946073047258080.  946073047258080. )
+BODY9999000_RADII          = (  100.  100.  100. )
 \begintext
 
 PINPOINT parameters for galaxy M82:
 \begindata
 SITES += ( 'SITE9999000' )
 SITE9999000_FRAME  = 'J2000'
 SITE9999000_IDCODE = 9999000
-SITE9999000_XYZ = ( -0.2975627098411663 0.17902917594015963 0.9377659558088944 )
+SITE9999000_XYZ = ( -97681008667696016.0 58769966492494512.0 307840738869915328.0 )
```

Comment:
This puts M82 at a nominal 100ly from the solar system barycenter, which is far enough to make parallax insignificant in observations.